### DISCUSSION TITLE
Tetris

### DISCUSSION THREAD

**Discussion by villares** [py5coding/py5generator#568] [Show and tell]
![image](https://github.com/user-attachments/assets/d4661a0d-aede-43b3-abc0-535c46158636)

Tetris in less than 130 lines of Python with py5, imported mode style.  :smile: 

https://github.com/villares/sketch-a-day/blob/main/2024/sketch_2024_07_09/sketch_2024_07_09.py

Tell me if you'd like me to make a module mode version, or if you want any parts explained.

**Comment by hx2A** [comment]
Tetris is a fun game. In high school I almost got suspended because I found Tetris hidden on a server and emailed it to everyone in the school. Unfortunately the disk quotas were not setup correctly (this was ~1990, when people didn't know about such things) so the whole computer system crashed. True story.
